es.json

```diff
@@ -1,134 +1,134 @@
+  "loader.starting": "Inicializando aplicación...",
+  "loader.realm.started": "Cargando reinos...",
+  "loader.realm.loaded": "Se encontraron {{size}} reinos: {{realms}}...",
+  "loader.realm.empty": "No hay reinos presentes. Intenta luego...",
+  "loader.realm.autoselected": "Se seleccionó '{{realm}}' como reino default...",
+  "loader.realm.failed": "Falla al cargar. Probablemente por mantenimiento de la aplicación...",
+  "loader.realm.scheduled": "Se intentó recargar reinos por {{time}} segundos...",
+  "loader.league.started": "Parseando ligas de '{{realm}}'...",
+  "loader.league.loaded": "Se encontraron {{size}} ligas: {{leagues}}...",
+  "loader.league.empty": "No se encontró ninguna liga. Probablemente por mantenimiento de la aplicación...",
+  "loader.league.autoselected": "Se seleccionó '{{league}}' como liga default...",
+  "loader.series.started": "Parseando series de '{{league}}'...",
+  "loader.series.loaded": "Se encontraron {{size}} mapas para '{{series}}' series...",
+  "loader.series.empty": "Series no presentes. Intentar luego...",
+  "loader.series.failed": "Error al cargar series. Probablemente por mantenimiento de la aplicación...",
+  "loader.series.scheduled": "Intentando recargar series después de {{time}} segundos...",
+  "login.hello": "Hola, {{username}}!",
+  "login.hero.selection_prompt": "Es momento de elegir tu personaje",
+  "login.not_logged": "No estás logueado.",
+  "login.loading": "Cargando...",
+  "login.hero.selection.hint": "Selecciona tu personaje. El juego destinará todos los mensajes a tu personaje activo, pero es mejor seleccionar tu personaje activo para simplificar a los jugadores el marcar que intercambio en la lista están completos.",  
+  "login.hero.selection.placeholder": "Selecciona tu personaje activo",
+  "login.error.credentials": "Error de autenticación",
+  "login.hero.empty.main": "Tus credenciales son correctas pero no encontramos ningún personaje para <b>{{username}}</b>.",
+  "login.hero.empty.warning.prefix": "Revisa que la <b>pestaña de personajes</b> no está oculta",
+  "login.hero.empty.warning.url_word": "aquí",
   "login.hero.empty.warning.suffix": ".",
-  "login.hero.empty.reminder": "Do not forget to save your changes there",
+  "login.hero.empty.reminder": "No olvides guardar tus cambios allí",
   "button.login.enter": "Enter",
-  "button.login.register": "Register",
-  "button.login.back": "Back",
-  "button.login.reload": "Reload",
-  "button.login.continue": "Continue",
-  "translation.info": "Want to improve the translation? Contribute here",
-  "contributors.text": "Translated by {{contributors.names}}",
+  "button.login.register": "Registrarse",
+  "button.login.back": "Volver",
+  "button.login.reload": "Recargar",
+  "button.login.continue": "Continuar",
+  "translation.info": "¿Quieres mejorar la traducción? Ayuda aquí",
+  "contributors.text": "Traducción por {{contributors.names}}",
   "contributors.names": ""
```

Comment:
Could you add your preferred name to the "contributors.names"?

Reply:
Done!